logger/logger.go

```diff
@@ -0,0 +1,30 @@
+package logger
+
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/threefoldtech/tf-pinning-service/config"
+)
+
+var Log *logrus.Logger
```

Comment:
this is not the logger is intended to be used. Instead your code can do something like 

```
import  (
  log "github.com/sirupsen/logrus"
)

func mycode() {
   log.Error() // or whatever
}
```
in your bin (main()) code then u just sett he proper logger configuration
```
log.SetFormatter() // and other logger config
```
and that's it, nobody need to share any gobals.
```